webservice/tests/test_webservice.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import mock
+from requests import exceptions
+
+from odoo.tests.common import tagged
+
+from odoo.addons.component.tests.common import SavepointComponentRegistryCase
```

Comment:
you don't need this class if you are not registering other components. Use SavepointComponentCase